src/app/routes/getInitialData/utils/getRouteProps/index.js

```diff
@@ -0,0 +1,25 @@
+import { matchRoutes } from 'react-router-config';
+import pathOr from 'ramda/src/pathOr';
+import { fallbackAmpParam, fallbackServiceParam } from './routeFallbackParams';
+
+const getRouteProps = (routes, url) => {
+  const matchedRoutes = matchRoutes(routes, url);
+
+  const route = pathOr(undefined, [0, 'route'], matchedRoutes);
+  const match = pathOr(undefined, [0, 'match'], matchedRoutes);
+  const params = pathOr({}, ['params'], match);
```

Comment:
if you choose to use ramda `path` then here you could do

`const params = path(['params'], match) || {};`

instead of importing 2 ramda functions. I don't see a problem with importing 2 ramda functions though!

Reply:
Idm importing 2 ramda functions, espesh as theyre small and we use them already elsewhere :)